From 33ffe432a94a5dad34cd930e6dbf363eff077550 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Wed, 17 Apr 2013 15:27:34 -0700
Subject: [PATCH 167/596] CLOUDERA-BUILD. Add changes to hadoop build to
 specify JDK1.7 as target (cherry picked from commit
 2564488ef8ac30c5edeb8706d0bd13ff145f6e33)

(cherry picked from commit 269c2ad6f2116530d408ec9ae2aaff54d303a1a1)

Conflicts:
	pom.xml

(cherry picked from commit be5d9b4c0dede984aced37beac3083cff2dcd0c5)
(cherry picked from commit a605dacffe4caea26552b9937a9a64cc3a853570)
(cherry picked from commit dd160c1211da733e6ae25e62321659e8724cf0aa)
---
 hadoop-project/pom.xml |    2 +-
 pom.xml                |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 43539fa..b0818bf 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -939,7 +939,7 @@
           <version>2.5.1</version>
           <configuration>
             <source>1.6</source>
-            <target>1.6</target>
+            <target>${targetJavaVersion}</target>
           </configuration>
         </plugin>
         <plugin>
diff --git a/pom.xml b/pom.xml
index f16f3c4..36c5b7d 100644
--- a/pom.xml
+++ b/pom.xml
@@ -110,6 +110,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <distMgmtStagingName>Apache Release Distribution Repository</distMgmtStagingName>
     <distMgmtStagingUrl>https://repository.apache.org/service/local/staging/deploy/maven2</distMgmtStagingUrl>
     <javaVersion>1.6</javaVersion>
+    <targetJavaVersion>1.6</targetJavaVersion>
 
     <!-- platform encoding override -->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-- 
1.7.9.5

